master/static/migrations/20211028145058_jobs.up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TYPE public.job_type AS ENUM (
+    'EXPERIMENT',
+    'NOTEBOOK',
+    'SHELL',
+    'COMMAND',
+    'TENSORBOARD'
+);
+
+CREATE TABLE public.jobs (
+    job_id text NOT NULL UNIQUE,
+    job_type job_type NOT NULL
+);
+
+
+BEGIN;
```

Comment:
there is no need as far as i can tell to nest a transaction here.

Reply:
is this because of the new "all migrations being treated as transactions" ? otherwise could it fails halfway? or would the go migrate take care of it somehow.

Reply:
yeah, this is because of that, all migrations are transactions now, so this is just a nested transaction.

Reply:
i approved but, i think we still need to remove this.

Reply:
removed